Modules/socketmodule.c

```diff
@@ -5315,7 +5331,10 @@ sock_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     if (new != NULL) {
         ((PySocketSockObject *)new)->sock_fd = INVALID_SOCKET;
         ((PySocketSockObject *)new)->sock_timeout = _PyTime_FromSeconds(-1);
-        ((PySocketSockObject *)new)->errorhandler = &set_error;
+        ((PySocketSockObject*)new)->errorhandler = &set_error;
+#ifdef MS_WINDOWS
+        ((PySocketSockObject*)new)->quickack = 0;
```

Comment:
```suggestion
        ((PySocketSockObject *)new)->errorhandler = &set_error;
#ifdef MS_WINDOWS
        ((PySocketSockObject *)new)->quickack = 0;
```

Seems no reason to change the existing style here.

Reply:
must have been a copy/paste auto-format from my editor